### ISSUE TITLE
Allow custom options

### ISSUE THREAD

**Issue by ChessSpider** [matthisk/django-chart#8]
Hi! It's me again.

I noticed not all options can be set from Python, e.g. `maintainAspectRatio` . 

Out of 
```javascript
    options: {
        responsive: true,
        maintainAspectRatio: false
    }
```

the responsive can be set, but not maintainAspectRatio (see See https://github.com/chartjs/Chart.js/issues/2958  ). Maybe others are missing as well.

Would it be an idea to add a custom_options dictionary which `def _get_options()` then merges with its own generated one?

Something like

```python
    custom_options = {}
   # .....
    def _get_options(self):
        option_keys = {'scales', 'layout', 'title', 'legend',
                       'tooltips', 'hover', 'animation', 'elements',
                       'responsive'}

        options = {key: self._get_options_attr(key)
                for key in option_keys if self._has_options_attr(key)}
         options.update(self.custom_options)
         return options
```

Using this solution, people can still easily set missing options if they're missing from the framework without having to rely on inheritance

**Comment by matthisk** [comment]
Yes I like this approach, this is something that is missing from the lib at the moment. Because I didn't have time to cross reference all Chart.JS options and only took the essential ones. Not sure if I like the name `custom_options` though, what if we would just call it `options`, what are your thoughts on this?

```python
class TimeSeriesChart(Chart):
    chart_type = 'line'
    scales = {
        'xAxes': [Axes(type='time', position='bottom')],
    }
    options = {
        'maintainAspectRatio': True
    }
```

Also we would have to decide on precedence of the `options` dict and the chart property config options, i.e. what happens if you set responsive to `False` in the `options` dict and to `True` on the Chart property. I would say the Chart property has precedence, this would also avoid any backwards compatibility issues (potentially people already defined a `options` property on their Chart subclasses since it isn't reserved in the current. 

If we decide on these issues you could make a PR for this feature, code wise it shouldn't be too complicated.

**Comment by ChessSpider** [comment]
Yes, I agree with you 100%.  Maybe you can just make the code change yourself without a PR.. it's not really worth the trouble at the moment to make a PR for this imho :)

```python
    options = {}
   # .....
    def _get_options(self):
        option_keys = {'scales', 'layout', 'title', 'legend',
                       'tooltips', 'hover', 'animation', 'elements',
                       'responsive'}
         options_tmp = self.options
         options_tmp.update( {key: self._get_options_attr(key)
                for key in option_keys if self._has_options_attr(key)} )
         return options_tmp
```

**Comment by matthisk** [comment]
Just released a new version (v0.4.0) with support for feature.
